test/workshop_test.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+// TestWorkshops searches the parent directory for directories that
+// contain a README.md file. It then parses those for START/END tags
+// and runs the bash scripts between those tags. It only supports bash
+// actions at this time.
+func TestWorkshops(tt *testing.T) {
+	workshops, err := findWorkshops()
+	if err != nil {
+		tt.Log("Error finding workshop READMEs")
+		tt.Fatal(err.Error())
+	}
+
+	for _, ws := range workshops {
+		text, err := ioutil.ReadFile(ws)
```

Comment:
why use `ReadFile` here instead of just passing the `os.File` as the reader for `GetActions`?